EFxceptions.Identity.SQLite.Tests/EFxceptions.Identity.SQLite.Tests.csproj

```diff
@@ -1,23 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
Try .NET 2.0 so it supports litterally everybody.

Reply:
@mabroukmahdhi did test with .NET 2.0 AND .NET 2.1 and both did not work with the library.